src/benchmark/benchmark_activeobjectmgr.cpp

```diff
@@ -105,7 +105,11 @@ void benchGetObjectsInArea(Catch::Benchmark::Chronometer &meter)
 TEST_CASE("ActiveObjectMgr") {
 	BENCH_INSIDE_RADIUS(200)
 	BENCH_INSIDE_RADIUS(1450)
+	BENCH_INSIDE_RADIUS(10000)
 
 	BENCH_IN_AREA(200)
 	BENCH_IN_AREA(1450)
+	BENCH_IN_AREA(10000)
```

Comment:
btw I didn't add it because it requires multiple gigabytes of RAM to run

Reply:
Huh? How so? I measured peak memory usage using `time -v` for `./bin/minetest --run-benchmarks --test-module ActiveObjectMgr` and got `Maximum resident set size (kbytes): 32232`, so around 32 MB. Why would 10k test objects (+ indexes on them) require multiple GB of RAM?

Reply:
I did mean the first one but looks like I remembered totally wrong.